aksel.nav.no/website/app/_ui/utils/format-text.ts

```diff
@@ -5,6 +5,26 @@ function capitalizeText(inputString: string): string {
   return inputString.charAt(0).toUpperCase() + inputString.slice(1);
 }
 
+/**
+ * Capitalize and space between each fragment
+ */
+const humanizeText = (str) =>
+  str
+    .split(/[-_\s]+/)
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
+    .join(" ");
+
+export const humanizeRedaksjonType = (type: string) => {
+  switch (type) {
+    case "miljoe":
+      return "Miljø";
+    case "team":
+      return "Team";
+    default:
+      return humanizeText(type);
```

Comment:
Since we can strictly type this based on Sanity-typegen, i don't really see a need for a fallback like this, and can probably just return null or "".